build_scripts/windows/azure-cli.wixproj

```diff
@@ -21,6 +12,15 @@
     <WixTargetsPath Condition=" '$(WixTargetsPath)' == '' AND '$(MSBuildExtensionsPath32)' != '' ">$(MSBuildExtensionsPath32)\Microsoft\WiX\v3.x\Wix.targets</WixTargetsPath>
     <WixTargetsPath Condition=" '$(WixTargetsPath)' == '' ">$(MSBuildExtensionsPath)\Microsoft\WiX\v3.x\Wix.targets</WixTargetsPath>
   </PropertyGroup>
+  <!-- Local WiX -->
+  <PropertyGroup>
+    <LocalWixRoot>artifacts\wix</LocalWixRoot>
+    <WixToolPath>$(MSBuildThisFileDirectory)$(LocalWixRoot)</WixToolPath>
+    <WixTargetsPath Condition="Exists('$(WixToolPath)\Wix.targets')">$(WixToolPath)\Wix.targets</WixTargetsPath>
+    <WixTasksPath Condition="Exists('$(WixToolPath)\wixtasks.dll')">$(WixToolPath)\wixtasks.dll</WixTasksPath>
```

Comment:
As we discussed offline, if `WixTasksPath` is a relative path, it is relative to `WixTargetsPath`, instead of the current file `build_scripts/windows/azure-cli.wixproj`. If we change `WixTasksPath` to `$(WixToolPath)\wixtasks.dll`, it appends it:

```
error MSB4062: The "HeatDirectory" task could not be loaded from the assembly C:\src\azure-cli\build_scripts\windows\artifacts\wix\artifacts\wix\wixtasks.dll
```

I think it is fine to keep relative path, as it makes the code succinct, doesn't it? It's the document https://wixtoolset.org/docs/v3/msbuild/daily_builds/#step-2-modify-your-wixproj-file that should be updated. 😉

```suggestion
    <WixToolPath>$(LocalWixRoot)</WixToolPath>
    <WixTargetsPath Condition="Exists('$(WixToolPath)\Wix.targets')">$(WixToolPath)\Wix.targets</WixTargetsPath>
    <WixTasksPath Condition="Exists('$(WixToolPath)\wixtasks.dll')">wixtasks.dll</WixTasksPath>
```

Reply:
If you look at WixToolPath now, it's absolute. And the condition and value should match. This works exactly as it should now without any guesswork.

Reply:
> And the condition and value should match.

This is a very good point. The fact that the `Condition` and the value don't match as they use different base path is indeed a little bit confusing.